app/views/dishes/_rating_star.html.haml

```diff
@@ -0,0 +1,9 @@
+- class_name = (!current_user || current_user == dish.user) ? 'readonly-rating-star' : 'rating-star'
```

Comment:
`current_user == dish.user` appears in 2+ places. Consider introducing a `Dish#owner?(user)` method so it'd become:

```ruby
dish.owner?(current_user)
```